platform/xbox/crt0.c

```diff
@@ -19,8 +19,6 @@ static int main_wrapper ()
 
 void XboxCRTEntry ()
 {
-    XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
```

Comment:
Should be able to also remove `#include <hal/video.h>` now

Reply:
Done!